docs/example-entity-definition.yml

```diff
@@ -49,3 +49,5 @@ goldenTags:
 configuration:
   # The amount of time without receiving telemetry before an entity is deleted. Defaults to EIGHT_DAYS
   entityExpirationTime: EIGHT_DAYS
+  # Defines whether the entities of this type should have an alert status associated, defaults to true
```

Comment:
not sure we want to call this `alert status`, that was the old name for the field. Should be `entity severity` now

Reply:
That's true, thanks! the field itself being called `alertable` still make sense right? (cc: @cianBuckley)

Reply:
`alertable` sounds good to me